go.mod

```diff
@@ -0,0 +1,14 @@
+module github.com/just-install/just-install
+
+require (
+	github.com/cheggaaa/pb v1.0.25
```

Comment:
Is there a reason why we are using both `github.com/cheggaaa/pb` and `gopkg.in/cheggaaa/pb.v1`?